test/conformance/ptp/ptp.go

```diff
@@ -364,6 +365,50 @@ var _ = Describe("[ptp]", func() {
 				Expect(slavePodDetected).ToNot(BeFalse(), "No slave pods detected")
 			})
 		})
+
+		Context("Running with fifo scheduling", func() {
+			BeforeEach(func() {
+				if discoveryFailed {
+					Skip("Failed to find a valid ptp slave configuration")
+				}
+
+				masterConfigs, slaveConfigs := discoveryPTPConfiguration(PtpLinuxDaemonNamespace)
+				ptpConfigs := append(masterConfigs, slaveConfigs...)
+
+				fifoPriorities = make(map[string]int64)
```

Comment:
You should add SCHED_FIFO by default to the ptpconfig , if not it can only work in discovery mode.
another option is to add an env variable to enable this as I did for PTP events @SchSeba  what are your thoughts on this?

Reply:
If I were to add SCHED_FIFO to the default config, and move the test to that section mode section, then this test case would only work in a non-SNO configuration, which is not ideal either.
At some point, we do need to clean this up so that all of the test cases support both discovery and non-discovery, and both SNO and multi-node, at which point this should be moved to that section, and fifo scheduling added.  In my opinion right now though, it is more important to be able to run in SNO than to be able to run in non-discovery mode.